x-pack/plugins/alerts/server/task_runner/task_runner.test.ts

```diff
@@ -28,11 +28,13 @@ import { IEventLogger } from '../../../event_log/server';
 import { SavedObjectsErrorHelpers } from '../../../../../src/core/server';
 import { Alert, RecoveredActionGroup } from '../../common';
 import { omit } from 'lodash';
+import { LicenseType } from '../../../licensing/server';
 const alertType = {
   id: 'test',
   name: 'My test alert',
   actionGroups: [{ id: 'default', name: 'Default' }, RecoveredActionGroup],
   defaultActionGroupId: 'default',
+  minimumLicenseRequired: 'basic' as LicenseType,
```

Comment:
**nit:** casting to `LicenseType` instead of defining the alert type as `: AlertType`.